packages/next/src/cli/next-build.ts

```diff
@@ -102,10 +101,11 @@ const nextBuild = (options: NextBuildOptions, directory?: string) => {
           err.code === 'EDGE_RUNTIME_UNSUPPORTED_API')
       ) {
         printAndExit(`> ${err.message}`)
-      } else {
-        console.error('> Build error occurred')
-        printAndExit(err)
+        return
       }
+
+      console.error('> Build error occurred')
+      printAndExit(err as string)
```

Comment:
I'm looking forward to more types and comments. But these kind of control-flow refactors  are not necessary and needlessly opinionated. All else being equal, code shouldn't change to make git-blame easier to navigate.